challenge.md

```diff
@@ -5,19 +5,26 @@
 Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+git checkout response
```

Comment:
This would change the currently active branch to the one called 'response'. Your goal is to download the repo and save it to your filesystem. You would achieve this by running this command:

`git clone <repo_url>`